data-8/jupyterhub-k8s
# Separate proxy

Closes #20 

Summary
-------
Added the separate hub / proxy for jhub.

How to Test
-------
1. Log into JupyterHub as user `test`
2. Delete hub deployment with `kubectl delete deployment hub-deployment`
3. Check Jupyter Notebook
12. Make sure that the notebook is still up for user `test`
